appinfo/info.xml

```diff
@@ -41,6 +41,7 @@ Developed with ❤️ by [LibreCode](https://librecode.coop). Help us transform
     <repository type="git">https://github.com/LibreSign/libresign</repository>
     <screenshot>https://raw.githubusercontent.com/LibreSign/libresign/main/img/screenshot/request-signature.png</screenshot>
     <screenshot>https://raw.githubusercontent.com/LibreSign/libresign/main/img/screenshot/presentation.gif</screenshot>
+	<donation>https://github.com/sponsors/LibreSign</donation>
```

Comment:
I saw that this is the default link (other) but also is possible to add link to PayPal and Stripe.

I created a link to Stripe:

https://buy.stripe.com/eVqfZibhx8QO3LseWc2kw00

Could you add this?

```suggestion
	<donation>https://github.com/sponsors/LibreSign</donation>
	<donation type="strype">https://github.com/sponsors/LibreSign</donation>
```

Reply:
Only a random comment:
It's curious that this element haven support to `lang` attribute.